src/argilla/client/feedback/dataset/local/mixins.py

```diff
@@ -234,7 +240,7 @@ def push_to_argilla(
         self: Union["FeedbackDataset", "ArgillaMixin"],
         name: str,
         workspace: Optional[Union[str, Workspace]] = None,
-        show_progress: bool = False,
+        show_progress: bool = True,
```

Comment:
this should be disabled by default

Reply:
@alvarobartt do we want to enable or disable this by default?

Reply:
Hmm not sure, I think there are some that have it enabled by default and some that have it disabled by default, it will depend on whether the dependencies we're using to display it are optional or not, but I think that enabled by default is the most compliant option with the former `rg.log`